outputFormat.js

```diff
@@ -149,13 +149,13 @@ function printWorld(property, center, currentlyDragging) {
 		outputTEXT.append("<p><b>Equal-area world map projections with poles represented as points</b></p>");
 		//loop through global data
 		for (var i = 0; i < 2; i++) {
-			outputTEXT.append("<p class='outputText'><span onmouseover='updateWorldMap(\"" + listWorld[i].projection + "\")'>" + listWorld[i].projection + stringLinks(listWorld[i].PROJ4, NaN, NaN, NaN, NaN, lng, NaN) + "<\span></p>");
+			outputTEXT.append("<p class='outputText'><span onmouseover='updateWorldMap(\"" + listWorld[i].projection + "\")'><span style='text-decoration: dashed underline;'>" + listWorld[i].projection + "</span>" + stringLinks(listWorld[i].PROJ4, NaN, NaN, NaN, NaN, lng, NaN) + "<\span></p>");
```

Comment:
I am not sure if I like this underlining... I suggest we remove it for now, but I am adding another line to the issues. I agree with you. They have to be somehow marked.

Reply:
agreed and removed them in the next 2 commits, thanks